src/index.ts

```diff
@@ -646,6 +648,9 @@ export default class Wallet {
   }
 }
 
+export default Wallet
+export { Wallet }
```

Comment:
If we would like to add a deprecation notice for the default export, it could look something like this:

```suggestion
/**
 * Please note that the default export for Wallet is DEPRECATED and
 * may be removed in the next major version.
 * Instead, please use the named export like so:
 * `import { Wallet } from 'ethereumjs-wallet'`
 *  or
 * `const { Wallet } = require('ethereumjs-wallet')`
 */
export default Wallet

export { Wallet }
```